optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -99,6 +102,8 @@
     @XStreamAlias("assertionScoreDirectorFactory")
     protected ScoreDirectorFactoryConfig assertionScoreDirectorFactory = null;
 
+    protected Boolean generateDroolsTestOnError = null;
```

Comment:
Just noticed one more thing, that this property isn't inherited in the inherit() method of this Config class? It should be if it's not.